auth/server/src/main/java/com/seagates3/dao/ldap/AuthStore.java

```diff
@@ -10,11 +10,11 @@
 interface AuthStore {
 
  public
-  void save(Map<String, Object> dataMap, Object obj,
+  void save(Map<String, Object> dataMap,
             String prefix) throws DataAccessException;
 
  public
-  Object find(String keyToFind, Object obj,
+  Object find(String keyToFind, Object obj, Object obj2,
```

Comment:
what is obj2, can you name it based on context

Reply:
obj2 can be any object here. So cant rename with specific context here

Reply:
can you mention example values passed as obj and obj2

Reply:
We can get this if we perform (ctrl + T) on method.